src/tests/unit/tests/electron/platform/android/rule-information-provider.test.ts

```diff
@@ -68,6 +68,13 @@ describe('RuleInformationProvider', () => {
         expect(unifiedResolution).toMatchSnapshot();
     });
 
+    test('getRuleInformation handles no foreground/background color values', () => {
+        const testRuleId: string = 'ColorContrast';
+        const ruleResult: RuleResultsData = buildColorContrastRuleResultObject('FAIL', 2.798498811425733, null, null);
```

Comment:
When the service has no colors, there's also no contrast ratio
```suggestion
        const ruleResult: RuleResultsData = buildColorContrastRuleResultObject('FAIL', null, null, null);
```

Reply:
I did not got any error from `'Color Contrast Ratio'` being null (I scan 3 or 4 real android apps) that's why that's not being changed here. I'd rather not change it as this scenario will quickly dissapear when we do filter any color contrast issue with confidence value None